fwbot/tests/test_repository.py

```diff
@@ -0,0 +1,62 @@
+from unittest.mock import Mock, patch
+
+from ..api import ForwardbotGitlabClient
+from ..models.repository import Repository
+from .common import ForwardBotCase, MockResponse
+
+
+class TestRepository(ForwardBotCase):
+    def test_get_branch_version(self):
+        self.assertEqual(1000.0, self.model_repo.get_branch_version("main"))
+        self.assertEqual(1000.0, self.model_repo.get_branch_version("master"))
+
+        self.assertEqual(12.0, self.model_repo.get_branch_version("12.0"))
```

Comment:
Usually it's:

`assertEqual(gotten, expected)`

Same in all other similar cases.